<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditFilterBase</class>
 <widget class="QWidget" name="AddEditFilterBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>141</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter &amp;Field:</string>
     </property>
     <property name="buddy">
      <cstring>filterField</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="filterField"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Filter &amp;Type:</string>
     </property>
     <property name="buddy">
      <cstring>filterType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="filterType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Text:</string>
     </property>
     <property name="buddy">
      <cstring>filterText</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KLineEdit" name="filterText"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Filter Action:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="filterAction"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="dontHideReplies">
     <property name="text">
      <string>Do not hide replies to me</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="KSeparator" name="kseparator"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
